misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/NotFoundButton.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import counterpart from 'counterpart';
+
+import ConfirmButton from './confirmButton/ConfirmButton';
```

Comment:
pls move it to common or to components.

Reply:
or better, move it in picking because that's the only place where we are using it.